Bug: `make state` failing

From one of our workshop testers:

```
npx ts-node bin/cdk_state.ts
npx: installed 8 in 1.29s
Cannot find module 'typescript'
make: *** [state] Error 1
```

--- a/cdk/Makefile
+++ b/cdk/Makefile
@@ -4,7 +4,7 @@
 bootstrap: | install deploy state create_grants
 
 install:
-	npm install
+	npm ci
 
 deploy:
 	npx cdk deploy BusyEngineersDocumentBucketStack --require-approval never

--- a/cdk/package.json
+++ b/cdk/package.json
@@ -18,9 +18,7 @@
     "aws-cdk": "^1.17.0",
     "jest": "^24.9.0",
     "prettier": "^1.19.1",
-    "ts-jest": "^24.0.2",
-    "ts-node": "^8.5.2",
-    "typescript": "~3.6.2"
+    "ts-jest": "^24.0.2"
   },
   "dependencies": {
     "@aws-cdk/aws-cloud9": "^1.17.0",
@@ -35,6 +33,8 @@
     "@iarna/toml": "^2.2.3",
     "@types/node": "^12.12.11",
     "source-map-support": "^0.5.16",
+    "typescript": "^3.7.2",
+    "ts-node": "^8.5.2",
     "untildify": "^4.0.0"
   }
 }

